source/_integrations/weatherflow.markdown

```diff
@@ -0,0 +1,61 @@
+---
+title: "WeatherFlow Local"
+description: "Instructions on how to integrate your WeatherFlow tempest into Home Assistant."
+ha_release: 2023.2.0
+ha_category: 
+  - Sensor
+ha_platforms:
+ - sensor
+ - enviornment
+ha_iot_class: "Local Polling"
+ha_config_flow: true
+ha_codeowners:
+  - '@natekspencer'
+  - '@jeeftor'
+ha_domain: weatherflow
+---
+
+## Overview
+
+This integration is a local-only WeatherFlow integration that reads UDP data from a [WeatherFlow Tempest](https://weatherflow.com/tempest-weather-system/) compatible weather stations. It utilizes WeatherFlow's published [UDP API](https://weatherflow.github.io/Tempest/api/udp.html) to decode and present the data.
+
+## Installation Notes
+
+When you add this integration to Home Assistant it will detect and decode any valid UDP packets sent from a valid weather station. Sensors will only be created in Home Assistant once it has received a valid reading for that particular sensor type.
+
+<div class='note'>
+It possible you will may see slight deviations between some of the values in your WeatherFlow app and Home Assistant. This only applies to sensors with calculated values, as there may be slight differences between how Home Assistant and WeatherFlow choose to calculate these values.
```

Comment:
Integration sensors should not calculate values. This comment, therefore, seems indicating the integration implementation isn't correct.

Reply:
@natekspencer  - Can you speak towards the calculations at all - are they coming from the backing library or are they done in the integration code?

@frenck  - if I recall correctly all the calculations should be done in the backing library - not in the integration correct?

Reply:
Home Assistant isn't doing the calculations. It might be more appropriate to say the reason there are differences is primarily due to the WeatherFlow app using cloud enhanced (neighbor stations/forecast/etc) data while local only reports information known to the local weather station only.